index.html

```diff
@@ -0,0 +1,143 @@
+                    <img id="hamburguer" class="hamburguer" src="./images/first_page/ham-icon.png" alt="Hamburger menu that opens the menu">
+                </div>
+                <div class="generalOptions">
+                    <div class="companiesLogo">
+                        <a href="index.html"><img class="logoIcon" src="./images/first_page/logo.jpg" alt=""></a>
+                        <p>Global Video Games</p>
+                    </div>
+                    <div class="sub-generalOptionsDivision">
+                        <ul class="sub-generalOptions">
+                            <li><a href="index.html">Home</a></li>
+                            <li><a href="main.html">About</a></li>
+                            <li><a href="#">Program</a></li>
+                            <li><a href="#">Join</a></li>
+                            <li><a href="#">Sponsor</a></li>
+                            <li><a href="#">News</a></li>
+                            <li><a href="#">CC Campaign</a></li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div id="mobile-nav" class="mobilenav hidden">
+            <p id="closeIcon" class="closeicon"><img src="./images/first_page/icn-close.svg" alt=""></p>
+            <ul>
+                <li id="home"><a href="index.html">Home</a></li>
+                <li id="aboutus"><a href="main.html">About Us</a></li>
+                <li><img src="./images/first_page/Rectangle.png" alt=""></li>
+                <li id="program"><a href="#">Program</a></li>
+                <li><img src="./images/first_page/Rectangle.png" alt=""></li>
+                <li id="join"><a href="#">Join</a></li>
+                <li><img src="./images/first_page/Rectangle.png" alt=""></li>
+                <li id="sponsor"><a href="#">Sponsor</a></li>
+                <li><img src="./images/first_page/Rectangle.png" alt=""></li>
+                <li id="news"><a href="#">News</a></li>
+                <li><img src="./images/first_page/Rectangle.png" alt=""></li>
+                <li id="campaign"><a href="#">CC Campaign</a></li>
+                <li><img src="./images/first_page/Rectangle.png" alt=""></li>
+            </ul>
+        </div>
+        <div class="mainpage container">
+            <h3 class="greetings">"Hello, Sharing World"</h3>
+            <h1 class="main">Creating Global Video Forums Regarding The News On Videogames</h1>
+            <p class="mainparagraph">An awesome reunion of videogame news sharing the information on them and how they can change your life on prices and by dedicating yourself to do so, how much are this companies given as rewards and what you need to do in order to attempt to be the new bigest gamer of all the times.</p>
+            <h2 class="mainsubtitl">2022.03.14(Monday) ~ 18(Friday)</h2>
+            <p class="mainparagraph2">@Play Station, Nintendo, Epic Game and more.</p>
+        </div>
```

Comment:
- [ ] Your main section's width is too large compared to the template. It shouldn't be this wide, but rather stop a bit after the half. Also, please make sure that you're matching the template in terms of height, your main section is really tiny compared to the template, it should take up the whole width of the screen.

| Your website  | Template |
| ------------- | ------------- |
| ![](https://i.imgur.com/m0c6NGb.png)  | ![](https://i.imgur.com/3Jby179.png)  |